content/swift/concepts/dictionaries/terms/contains/contains.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.contains'
+Description: 'a Boolean value that indicates whether a dictionary contains a specified key-value pair.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Dictionaries'
+  - 'Properties'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.contains`** method returns a Boolean value that indicates whether a dictionary contains a specified key-value pair. The method takes a single argument, which is a tuple that contains a key-value pair. If the dictionary contains the specified key-value pair, the method returns `true`; otherwise, it returns `false`.

```

Comment:
``` suggestion
The **`.contains`** method returns a Boolean value that indicates whether a dictionary contains a specified key. The method takes a single argument, which is a key. If the dictionary contains the specified key, the method returns `true`; otherwise, it returns `false`.

```
The contains method typically checks for the existence of a key not for the existence of a key-value pair, so it does not take any tuple containing the key-value pairs as an argument, it checks for a key existence. 

Also to check the existence of a key-value pair would require a custom closure. Please read more about this so you can understand it better.